scripts/pylint_extensions_test.py

```diff
@@ -90,6 +91,40 @@ def test(test_var_one, test_var_two=4, test_var_three=5, test_var_four="test_che
                 func_call_node_three)
 
 
+class NonExplicitKeywordArgsCheckerTests(unittest.TestCase):
```

Comment:
I highly doubt that this test will pass since there is no class named `NonExplicitKeywordArgsCheckerTests` (the check has been added to the existing `ExplicitKeywordArgsCheckerTests`).

Reply:
The test does work because the actual checker is being assigned inside the class itself.